client/landing/stepper/declarative-flow/internals/steps-repository/pattern-assembler/pattern-assembler-preview.tsx

```diff
@@ -65,7 +66,15 @@ const PatternAssemblerPreview = ( { header, sections = [], footer, scrollToSelec
 						) }
 					>
 						{ ! hasSelectedPatterns && (
-							<span>{ translate( 'Your page is blank. Start adding content on the left.' ) }</span>
+							<>
+								<Icon className="pattern-assembler-preview__icon" icon={ layout } size={ 100 } />
```

Comment:
I saw the size is 48px in 7l53h5fxAUNjVRBQ82YFwn-fi-1407%3A56582&t=A4AadfvfXDygBpKk-0